packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

```diff
@@ -70,8 +69,8 @@ The sample app uses the default scopes provided in the SDK, which include `openi
 
 After you complete the configurations, run the sample app:
 
-* [Run the SDK sample app](/docs/guides/oie-embedded-common-run-samples/-/main/)
-* [Run the Widget sample app](/docs/guides/oie-embedded-common-run-samples/-/main/)
+* [Run the SDK sample app](/docs/guides/oie-embedded-common-run-samples/-/main/#run-the-sdk-sample-app)
+* [Run the Widget sample app](/docs/guides/oie-embedded-common-run-samples/-/main/#run-the-widget-sample-app)
```

Comment:
Suggest the sentence before the bullet list to be:

After you complete the configurations:
* Run the ...
* Run the ...